app/_src/gateway/install/linux/amazon-linux.md

```diff
@@ -63,11 +105,41 @@ sudo yum install -y kong-{{page.versions.ce}}.rpm
 {{ install_package | indent | replace: " </code>", "</code>" }}
 
 {% endnavtab %}
-{% navtab YUM repository %}
+{% navtab YUM repository (AL2023) %}
+
+Install the YUM repository from the command line.
+
+1. Download the Kong APT repository:
+    ```bash
+    curl https://download.konghq.com/gateway-3.x-amazonlinux-2023/config.repo | sudo tee /etc/yum.repos.d/kong.repo
```

Comment:
This needs to match the (Amazon Linux 2) "Download the Kong YUM repository" instructions below.